apps/src/netsim/NetSimVisualization.js

```diff
@@ -945,8 +957,12 @@ NetSimVisualization.prototype.setEncodings = function(newEncodings) {
  * Kick off an animation that will show the state of the simplex wire being
  * set by the local node.
  * @param {"0"|"1"} newState
+ * @param {boolean} fromRemote - if true, animate the wire being set from the remote node
  */
-NetSimVisualization.prototype.animateSetWireState = function(newState) {
+NetSimVisualization.prototype.animateSetWireState = function(
+  newState,
+  fromRemote
```

Comment:
Ah, but now I understand how `fromRemote` may be useful for visualizing the change.  Hmm...